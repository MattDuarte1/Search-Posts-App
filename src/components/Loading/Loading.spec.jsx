import { render } from '@testing-library/react';
import { Loading } from '.';

it('should match snapshot', () => {
  const { container } = render(<Loading />);

  expect(container.firstChild).toMatchSnapshot();
});
