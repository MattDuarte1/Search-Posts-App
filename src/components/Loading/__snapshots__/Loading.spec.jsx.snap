// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
<div
  class="loading-container"
>
  <div
    class="spinner"
  />
  <h1>
    loading
  </h1>
</div>
`;
